src/MultiFormGoals/ProgressBar/Query.php

```diff
@@ -25,29 +25,21 @@ public function __construct( $formIDs ) {
 	public function getSQL() {
 		$sql = "
             SELECT
-                sum( meta_value ) as total,
-                count( meta_value ) as count,
-                avg( meta_value ) as average
+                sum( revenue.amount ) as total,
+                count( payment.ID ) as count
             FROM wp_posts as payment
-                JOIN wp_give_donationmeta as meta
-                    ON meta.donation_id = payment.ID
+                JOIN wp_give_revenue as revenue
+                    ON revenue.donation_id = payment.ID
             WHERE
                 payment.post_type = 'give_payment'
                 AND
-                meta.meta_key = '_give_payment_total'
-                AND
-                payment.post_status = 'publish'
+                payment.post_status IN ( 'publish', 'give_subscription' )
```

Comment:
@kjohnson I think we can add a method in `Revenue` repository class to add `post_type` and `post_type` in the SQL query. I can see that will be helpful for most of the SQL queries in the revenue table.